probeservices/checkin_test.go

```diff
@@ -0,0 +1,66 @@
+package probeservices_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/ooni/probe-engine/model"
+)
+
+func TestCheckInSuccess(t *testing.T) {
+	client := newclient()
+	client.BaseURL = "https://ams-pg-test.ooni.org"
+	config := model.CheckInConfig{
+		Charging:        true,
+		OnWiFi:          true,
+		Platform:        "string",
+		ProbeASN:        "string",
+		ProbeCC:         "string",
+		RunType:         "string",
+		SoftwareVersion: "string",
+		WebConnectivity: model.CategoryCodes{
+			CategoryCodes: []string{"NEWS", "CULTR"},
+		},
+	}
+	ctx := context.Background()
+	result, err := client.CheckIn(ctx, config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	//TODO what do we want to check?
+	if len(result.WebConnectivity.URLs) < 1 {
```

Comment:
This needs to check for `result.WebConnectivity == nil` once we make this field nullable.